elements/storytelling/src/markdown-it-plugin/markdown-it-decorate-improved.js

```diff
@@ -72,15 +74,30 @@ function curlyAttrs(state) {
         stack,
         nav,
         finalTokens,
-        sectionStartIndex
+        sectionStartIndex,
+        state
       );
 
+    if (
+      token.type === "heading_open" &&
+      token.tag === "h1" &&
```

Comment:
Seeing that in https://github.com/EOX-A/EOxElements/issues/687 we identified `h2` tags suitable for sections, I think it would make sense to use `h2` for this as well, seeing that we mainly will use this functionality to replace sections with custom elements.

Reply:
Another thought, wouldn't it be good to allow entirely replacing the default rendered element with any element, not just custom elements?

Example: 
```md
## Hello <!--{as="p" .custom-paragraph}--> // would replace h1 with a p tag
```

Reply:
Done ✅